README.md

```diff
@@ -44,7 +44,7 @@ fish install --path=~/.local/share/omf --config=~/.config/omf
 You can verify the integrity of the downloaded installer by verifying the script against [this checksum](bin/install.sha256):
 
 ```
-bb1f4025934600ea6feef2ec11660e17e2b6449c5a23c033860aed712ad328c9 install
+429a76e5b5e692c921aa03456a41258b614374426f959535167222a28b676201 install
```

Comment:
This was missed since the last changes in the `install` script and will be missed again. So better to only include the SHA in one place. There said, I'd rather have it like the following:

```suggestion
curl -sL https://raw.githubusercontent.com/oh-my-fish/oh-my-fish/master/bin/install.sha256 | shasum -a 256 --check
```